gammapy/irf/psf_gauss.py

```diff
@@ -61,6 +61,10 @@ class EnergyDependentMultiGaussPSF:
         psf = EnergyDependentMultiGaussPSF.read(filename, hdu='POINT SPREAD FUNCTION')
         psf.plot_containment(0.68, show_safe_energy=False)
         plt.show()
+
```

Comment:
This example could be just changed to read `"$GAMMAPY_DATA/cta-1dc/caldb/data/cta/1dc/bcf/South_z20_50h/irf_file.fits"` instead...